libraries/provider_firewall_windows.rb

```diff
@@ -57,6 +57,14 @@ def whyrun_supported?
         end
       end
 
+      # If logging rules are not provided, then create a rule to disable them (windows default)
+      unless new_resource.rules['windows'].key?('set currentprofile logging droppedconnections enable')
```

Comment:
Shouldn't we do nothing if there aren't existing logging rules provided? I don't want to maintain Windows defaults in this cookbook long term.